tests/bplist_regen.sh

```diff
@@ -36,6 +36,7 @@ find . -mindepth 1 \
 echo ./bob/Blueprints >> "${TEMP_LIST_FILE}"
 echo ./bob/blueprint/Blueprints >> "${TEMP_LIST_FILE}"
 
+LC_ALL=C
```

Comment:
Does this need to be exported?

On my machine, I don't have `LC_ALL` set at all (i.e. it's not already exported but set to another value).

This means if I run `bplist_regen.sh`, even with this change, it gets ordered strangely, with `Blueprints` after `blueprint`.

Changing to `export LC_ALL=C` fixes this.

Reply:
Alternatively, just prefix the sort command with this (it should be the only relevant commant that needs it)

Reply:
I've updated the patch to prefix the command. I don't think the script should export this.